app/code/Magento/Ui/view/base/web/js/grid/columns/image-preview.js

```diff
@@ -13,6 +13,7 @@ define([
         defaults: {
             bodyTmpl: 'ui/grid/columns/image-preview',
             previewImageSelector: '[data-image-preview]',
+            adobeStockModalSelector: '.modal-slide.adobe-stock-modal',
```

Comment:
can the selector (and property) be modified not to mention adobe stock (as that is not currently adobe stock scope)?

Reply:
Fixed this too, thanks!